### ISSUE TITLE
Disaster Resilience: Card B - Significant Structural Damage

### ISSUE THREAD

**Issue by PoeStewart** [hackoregon/civic#253]
# Story Card Request

**Project:**
Disaster Resilience

**Card Title:**
Significant Structural Damage

## Description / Wireframe
This card shows key pieces of infrastructure (river bridges across the Willamette, airport, fuel storage) that are estimated to be destroyed or damaged beyond a usable state, using a total deformation wet scenario. Total Deformation (average of liquefaction/landslide combined) during a wet season is impacted by both landslide potential and liquefaction potential. Landslide potential is based on topography and soil conditions. Liquefaction potential estimates the potential for saturated (wet), unconsolidated/loose ground materials to liquefy or give way during an earthquake.

## Data Visualization / Design Concerns
We want some way to show how things will be impacted without having any specific data to visualize beyond perhaps a bar graph showing that only 2 out of ten bridges will be functional. 

![significant structural damage](https://user-images.githubusercontent.com/17058320/41373031-f6bd0ca6-6f03-11e8-89a2-b470778f315a.PNG)

https://youtu.be/sn98JkN5HXc

## Context
This card sums up some of the most pressing concerns facing Portland after an earthquake with an emphasis on prevention. 

### See the source

http://www.oregongeology.org/pubs/ofr/p-O-18-02.htm

**Comment by PoeStewart** [comment]
@eecooper What do you think of this for Card #2 or B?

**Comment by eecooper** [comment]
So, simplified by dropping the map and the bridge-graph; what breaks up the text is the table/infographic summary (correct?).  
Do you think we should add a #4 for URM?  It definitely fits with the card but might be too much info.

**Comment by PoeStewart** [comment]
I thought maybe that's the Impact Tweet?

**Comment by eecooper** [comment]
Ah, I see, that's why it's alone at the bottom.  Looks good to me!

**Comment by eecooper** [comment]
Is that photo image sufficient for this card to move forward?

**Comment by PoeStewart** [comment]
Not sure, I'm gonna say yes and toss it to Jaron and if he doesn't like it I assume he will let us know :)

**Comment by PoeStewart** [comment]
@jaronheard I think this one is ready as it doesn't necessarily have any data attached to it. It does Have some possible graphical elements (Airport, bridges, tanks - see above sketch), do we need to provide those?

**Comment by jaronheard** [comment]
Yep, marked this as ready. I've asked @jasongiglio to work on the infographic, but in case he can't, here are the instructions I provided:
Just use some icons from the noun-project and lay them out with text in an SVG. Would make them so that they can be layed out three across on desktop and stack on mobile. Be sure to use colors and typography from the style guide.

**Comment by jasongiglio** [comment]
Here are 3 SVGs for the "infographic". Wasn't sure in bar graph color so I used shat I think is HO brand color.

**Comment by jasongiglio** [comment]
Yes, which one?

**Comment by jasongiglio** [comment]
How's this: https://drive.google.com/open?id=1wrwWqoHffE4MBxfsuR6wGSt0YTBE9xtJ

**Comment by jasongiglio** [comment]
@jaronheard Here are the final infographics: https://drive.google.com/open?id=1Y35XvirpZPTjEj29Bzp4MK_CisFTlyIw
